cmd/crio/main.go

```diff
@@ -342,6 +342,13 @@ func main() {
 			}
 		}
 
+		// We always use "Volatile: true" to create containers, which means that we might 
```

Comment:
I don't believe `Volatile: true` is a default value. can this be reworded to 
```
In situations where "Volatile: true" is specified in the container storage, there are containers and layers
that may be lost track of in an unclean shutdown. 
We need to call the garbage collection function to clean up the redundant files.
```

Reply:
I have modified it according to your requirement. But what I can see from the code is that `Volatile: true` is a fixed value and always true.
https://github.com/cri-o/cri-o/blob/e46a5207f2a05ad224b98dfc49f77fb996bb30ce/internal/storage/runtime.go#L260-L267

Reply:
ah I had the wrong option in mind (I was thinking they were for a transient store, not volatile containers)
you are correct, we are always setting volatile, I think you can revert to what it was before. sorry!

Reply:
> ah I had the wrong option in mind (I was thinking they were for a transient store, not volatile containers) you are correct, we are always setting volatile, I think you can revert to what it was before. sorry!

That's not a problem. I'm not very confident in my English. 
I trust your judgment and I will revert it if you think it was better before. It's easy and doesn't take much time.